composer.json

```diff
@@ -3,14 +3,14 @@
     "description": "Wrapper for League of Legends API",
     "require": {
         "guzzlehttp/guzzle": "^6.0",
-        "doctrine/cache": "^1.6",
-        "predis/predis": "~1.0"
+        "symfony/cache": "^3.2"
     },
     "keywords": [
         "lol", "api" , "league of legends", "lol api", "library"
     ],
     "require-dev": {
-        "phpunit/phpunit": "^5.2"
+        "phpunit/phpunit": "^5.2",
+        "predis/predis": "~1.0"
```

Comment:
Why moving it to require-dev ? If we want to suggest it, composer.json has a section for that